py_clob_client/client.py

```diff
@@ -225,14 +228,24 @@ def get_price(self, token_id, side):
         """
         return get("{}{}?token_id={}&side={}".format(self.host, PRICE, token_id, side))
 
+    def get_tick_size(self, token_id: str) -> TickSize:
+        if token_id in self.__tick_sizes:
+            return self.__tick_sizes[token_id]
+
+        result = get("{}{}?token_id={}".format(self.host, GET_TICK_SIZE, token_id))
+        self.__tick_sizes[token_id] = result["minimum_tick_size"]
+
+        return self.__tick_sizes[token_id]
+
     def create_order(self, order_args: OrderArgs):
         """
         Creates and signs an order
         Level 2 Auth required
         """
         self.assert_level_2_auth()
 
-        return self.builder.create_order(order_args)
+        tick_size = self.get_tick_size(order_args.token_id)
```

Comment:
let's allow `tick_size` to be an optional parameter and if it isn't passed make the request to get the market's tick size, otherwise we can forgo the request

Reply:
New parameter added ✅